### ISSUE TITLE
Feature to add a custom theme

### ISSUE THREAD

**Issue by rishiraj824** [fossasia/chat.susi.ai#233]
**Actual Behaviour**

Currently, one can select a pre-defined theme but cannot create a custom theme

**Expected Behaviour**

One should be able to set a custom color in the Chat.
One should also be able to save a theme https://github.com/fossasia/chat.susi.ai/issues/192

**Would you like to work on the issue?**

Anyone is free to take this up.

**Comment by uday96** [comment]
So creating a custom theme as in the user must be given an option to choose styling for everything?

The header background
Font color
document background etc?

And must also be given an option to preview a static page or so with the selected option?
And then when he is satisfied, click on submit to store that as a custom theme?

**Comment by rishiraj824** [comment]
@uday96 Yes, all the changes which are currently happening while switching themes. 
Maybe a dialog can be used to let users choose the color for each segment. For a preview, the user can see the changes directly as the color is changed.

**Comment by rishiraj824** [comment]
@uday96 I can work on this partly 
- To create a dialog for changing themes.
- Preview the theme directly.

**Comment by uday96** [comment]
@rishiraj824 sure :+1:
